spark/src/main/resources/R/zeppelin_sparkr.R

```diff
@@ -31,7 +31,7 @@ print(paste("LibPath ", libPath))
 library(SparkR)
 
 
-SparkR:::connectBackend("localhost", port, 6000)
+SparkR:::connectBackend("localhost", port)
```

Comment:
This is for supporting spark 2.1, remove this would cause sparkr break in spark 2.1

Reply:
to be clear, this was never an actual public API in Spark. This is an internal private method and that is being called here.

but in any case, I'm not sure I follow why the setting can't be set by setting `SPARKR_BACKEND_CONNECTION_TIMEOUT`

`connectionTimeout <- as.numeric(Sys.getenv("SPARKR_BACKEND_CONNECTION_TIMEOUT", "6000"))`

Reply:
any update regarding this?